consonance-arch/src/test/java/io/consonance/arch/test/TestWorkerWithMocking.java

```diff
@@ -1,281 +0,0 @@
-/*
```

Comment:
This class will come back, just removed it to see if I could sidestep the classloader issue I ran into